<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDialog</class>
 <widget class="QDialog" name="ExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Choose the way to export</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QPushButton" name="selectionButton">
        <property name="text">
         <string>Selection</string>
        </property>
        <property name="icon">
         <iconset resource="../../../3rdparty/icons.qrc">
          <normaloff>:/icons/backup/view-media-playlist.svg</normaloff>:/icons/backup/view-media-playlist.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="bookButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Book</string>
        </property>
        <property name="icon">
         <iconset resource="../../../3rdparty/icons.qrc">
          <normaloff>:/icons/backup/address-book-new.svg</normaloff>:/icons/backup/address-book-new.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="bookDetailLabel">
        <property name="text">
         <string>Export a book or a chapter using &quot;Section&quot; items</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="selectionDetailLabel">
        <property name="text">
         <string>Select the items to export</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>bookButton</tabstop>
  <tabstop>selectionButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../../3rdparty/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
